ui/src/pageLayout/components/NavMenuSubItem.tsx

```diff
@@ -8,9 +8,10 @@ import {NavMenuType} from 'src/clockface'
 
 interface PassedProps {
   title: string
-  path: string
+  path?: string
   active: boolean
   className?: string
+  onClick?: () => void
 }
 
 interface DefaultProps {
```

Comment:
We could update this to the new DefaultProps pattern. If we just merge the interface with the `PassedProps` interface and don't specify the type for defaultProps TypeScript should be able to infer the default props.